backend/src/test/resources/queries/add-person

```diff
@@ -25,7 +25,10 @@ mutation createPersonEntry(
     city: $city
     state: $state
     zipCode: $zipCode
-    telephone: $telephone
+    phoneNumbers: [{
+      number: $telephone
+      type: "MOBILE"
```

Comment:
i don't think we put literals in these test query templates anywhere else, can we pass in the phoneType in tests instead of hardcoding it here?

Reply:
You betcha: https://github.com/CDCgov/prime-simplereport/pull/1532/commits/0b733c0f4620274b3e908335fad6d76650cb85ad#diff-1aea0103a60315d5a9925f2760388195e2c4e884e89794d9005bedaf66e3c476R279-R297